modules/juce_audio_basics/buffers/juce_FloatVectorOperations.cpp

```diff
@@ -1033,9 +1033,11 @@ intptr_t JUCE_CALLTYPE FloatVectorOperations::getFpStatusRegister() noexcept
     intptr_t fpsr = 0;
   #if JUCE_INTEL && JUCE_USE_SSE_INTRINSICS
     fpsr = static_cast<intptr_t> (_mm_getcsr());
-  #elif defined (__arm64__) || defined (__aarch64__) || JUCE_USE_ARM_NEON
+  #elif defined (__arm64__) || defined (__aarch64__) || defined (_M_ARM64) || JUCE_USE_ARM_NEON

```

Comment:
The `_M_ARM64` is redundant because you're defining [`JUCE_USE_ARM_NEON`](https://github.com/juce-framework/JUCE/pull/668/files#diff-49c75ab597f7147dafdc5fbe6abe4580R90).